.aliases

```diff
@@ -37,9 +37,10 @@ alias gpull='git rev-parse --abbrev-ref HEAD | xargs git pull origin'
 alias gaddnew='git status --short | grep -v "??" | cut -d " " -f 3 | xargs git add'
   alias gtnew='gaddnew'
   alias gtn='gaddnew'
-alias gd='git diff'
+# Consider using this with gcheck as well?
+alias gd='git status -s | fzf --no-sort --reverse --preview "git diff --color=always {+2}" --preview-window=right:60%:wrap --bind=ctrl-k:preview-up --bind=ctrl-j:preview-down'
```

Comment:
This is great, but it could end up feeling a bit clunky if it needs to be used too often. It may be good idea to include a simpler diff alias as well for quick checks.